.gitmodules

```diff
@@ -1,9 +1,6 @@
 [submodule "lib/forge-std"]
 	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
-[submodule "lib/solmate"]
```

Comment:
ahh, that because I was looking to override `_beforetokentransfer` function for soulbound, didn't find it in the solmate contract, so removed it even before checking OZ contracts lol. Realised later that it has been deprecated in favour of `_update`

Reply:
gotcha, cool we can continue with OZ for the time being.